build.bat

```diff
@@ -0,0 +1,28 @@
+@echo off
+
+echo %USERNAME%
+
+if exist "C:\Program files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build" (
+    set "VCVARALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build"
+)
+if exist "C:\Program files (x86)\Microsoft Visual Studio\2017\Professional\VC\Auxiliary\Build" (
+    set "VCVARALLDIR=C:\Program files (x86)\Microsoft Visual Studio\2017\Professional\VC\Auxiliary\Build"
+)
+
+call "%VCVARALLDIR%\vcvarsall.bat" x64
+
+REM Building the Beckhoff Builder
+call msbuild.exe /p:Configuration=Release;Platform=x64 util_scripts/Builder/BeckhoffBuilder.sln
+
+if %ERRORLEVEL% neq 0 goto :PROBLEM
+
+call .\util_scripts\Builder\bin\x64\Release\BeckhoffBuilder.exe "%~dp0\PLC Development.sln"
```

Comment:
I think this is ok. For the time being we will only have one project to build and if we change the name it will just involve a small change here and won't require a re-compile of the builder.